src/orchestrator/orchestrator.rs

```diff
@@ -103,58 +131,76 @@ impl<const N_ASSETS: usize, const N_BYTES: usize> Orchestrator<N_ASSETS, N_BYTES
             // - Parses CSV file into 'entries'.
             // - Sends 'entries' to executors via [entries_tx] channel.
             //
-            let (start, end) = self.calculate_task_range(i, number_executor);
+            let (start, end) = self.calculate_task_range(i, executor_count);
             let entry_csvs_slice = self.entry_csvs[start..end].to_vec(); // Clone only the necessary slice
 
+            // let stop_tx_clone = stop_tx.clone();
```

Comment:
Seems to be a leftover

Reply:
Removed at [3e39b10](https://github.com/summa-dev/summa-aggregation/pull/3/commits/3e39b105ff25c1ea2d7e99a187c6c59eff62a6c3)